src/util/tags.ts

```diff
@@ -66,6 +66,22 @@ function tagify(
         editTags,
         delimiters,
         whitelist: whitelistTransformed,
+        templates: {
+            tag(tagData: TagRecord) {
+                // Default template without title to prevent the default tag tooltip from showing.
+                return `<tag 
+                    contenteditable='false' 
+                    spellcheck='false' 
+                    tabIndex="${this.settings.a11y.focusableTags ? 0 : -1}"
+                    class="${this.settings.classNames.tag}"
+                    ${this.getAttributes(tagData)}> 
+                        <x title='' class="${this.settings.classNames.tagX}" role='button' aria-label='remove tag'></x>
+                        <div>
+                            <span class="${this.settings.classNames.tagText}">${tagData[this.settings.tagTextProp] || tagData.value}</span>
+                        </div>
+                    </tag>`;
+            },
```

Comment:
No inline HTML

Reply:
i might need help since i'm not that familiar with type script.
As far as i can see i cant use `renderTemplate` since an async function can't be assigned to `tag: string`.
I could use `createElements`, but then i have to either extend the function to allow custom tags and attributes or change the custom tags to html compliant tags and adjust the styling.